docs/source/conf.py

```diff
@@ -47,12 +47,12 @@
 # If so, generate a config.rst file and populate it with documentation about
 # configuration options
 
-if os.environ.get('READTHEDOCS', ''):
+# if os.environ.get('READTHEDOCS', ''):
 
-    # Readthedocs doesn't run our Makefile, so we do this to force it to generate
-    # the config docs.
-    with open('../autogen_config.py') as f:
-        exec(compile(f.read(), '../autogen_config.py', 'exec'), {})
+#     # Readthedocs doesn't run our Makefile, so we do this to force it to generate
+#     # the config docs.
+#     with open('../autogen_config.py') as f:
+#         exec(compile(f.read(), '../autogen_config.py', 'exec'), {})
```

Comment:
Should it stay or should it go? :)

Reply:
Decided to keep this in place. I just temporarily commented it out :)